src/Ryujinx.Ui.Common/Helper/ShortcutHelper.cs

```diff
@@ -128,10 +128,7 @@ public static void CreateAppShortcut(string applicationFilePath, string applicat
         private static string GetArgsString(string basePath, string appFilePath)
         {
             // args are first defined as a list, for easier adjustments in the future
-            var argsList = new List<string>
-            {
-                basePath,
-            };
+            var argsList = new List<string>();
```

Comment:
How do you want to launch Ryujinx without adding the path to the binary?
```suggestion
            var argsList = new List<string>
            {
                basePath,
            };
```